pkg/entry/mongo/document_test.go

```diff
@@ -126,4 +126,54 @@ var _ = Describe("Convert document", func() {
 			Entry("platform ID", mongo.PlatformIDKey, false),
 		)
 	})
+
+	Context("With \\n end of logs", func() {
+		var entry map[interface{}]interface{}
+
+		BeforeEach(func() {
+			entry = map[interface{}]interface{}{
+				mongo.StreamKey:         stringEntry("stream"),
+				mongo.TimeKey:           timeEntry(time.Now()),
+				mongo.AppIDKey:          stringEntry("appID"),
+				mongo.AppExecutionIDKey: stringEntry("appExecutionID"),
+				mongo.ContainerIDKey:    stringEntry("containerID"),
+				mongo.ProjectIDKey:      stringEntry("projectID"),
+				mongo.CustomerKey:       stringEntry("customer"),
+				mongo.PlatformIDKey:     stringEntry("platformID"),
+			}
+		})
+
+		It("Should remove \\n from log end", func() {
+			entry[mongo.LogKey] = stringEntry("log\n")
+
+			d, err := mongo.Convert(ctx, time.Now(), entry)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(d).ToNot(BeNil())
+			Expect(d).To(BeAssignableToTypeOf(&mongo.AppLogDocument{}))
+			document := d.(*mongo.AppLogDocument)
+			Expect(document.Log).To(BeEquivalentTo(stringEntry("log")))
+		})
+
+		It("Should remove multiple \\n from log end", func() {
+			entry[mongo.LogKey] = stringEntry("log\n\n\n")
+
+			d, err := mongo.Convert(ctx, time.Now(), entry)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(d).ToNot(BeNil())
+			Expect(d).To(BeAssignableToTypeOf(&mongo.AppLogDocument{}))
+			document := d.(*mongo.AppLogDocument)
+			Expect(document.Log).To(BeEquivalentTo(stringEntry("log")))
+		})
```

Comment:
You sure that is the expected behavior?

Reply:
Not really, It was an unexpected behavior. Using a regexp, it was easy to protect against this unexpected event. I remove it